content/en/continuous_integration/setup_pipelines/customspans.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Custom spans
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/setup_pipelines/customspans/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+Custom spans provide a way to trace individual commands in your CI pipelines, allowing you to measure the time your command takes without taking into account any setup or teardown actions that the job might have (for example, downloading Docker images or waiting for an available node in a Kubernetes-based infrastructure). These spans appear as part of the pipeline's trace:
+
+{{< img src="ci/ci-custom-spans.png" alt="Details for a single pipeline with custom spans" style="width:100%;">}}
+
+## Compatibility
+
+Custom spans work with the following CI providers:
+
+- Jenkins with Datadog plugin >= v3.2.0
+- CircleCI
+
+## Installing the Datadog CI CLI
+
+Install the [`datadog-ci`][1] CLI globally using `npm`:
+
+{{< code-block lang="bash" >}}
+npm install -g @datadog/datadog-ci
+{{< /code-block >}}
+
+## Tracing a command
+
+To trace a command, run the following:
+
+{{< code-block lang="bash" >}}
+datadog-ci trace --name <name> <command>
+{{< /code-block >}}
+
+Specify a valid [Datadog API key][2] in the `DATADOG_API_KEY` environment variable. For example:
+
+{{< site-region region="us" >}}
+{{< code-block lang="bash" >}}
+DATADOG_API_KEY=<api_key> datadog-ci trace \
+  --service "Greet" \
+  echo "Hello World"
```

Comment:
The block above this one says:

>  To trace a command, run the following:
> ```
> datadog-ci trace --name <name> <command>
> ```

Should the examples also include `--name` for consistency?